Jenkinsfile.eclipse

```diff
@@ -60,8 +60,6 @@ pipeline {
             steps {
                 // build antenna and also deploy it to an output repository
                 sh """
-                    rm -rf repository
-                    mkdir -p repository
                     mvn -Dmaven.repo.local=\$(readlink -f localRepository) \
```

Comment:
These statements have at least the effect that an already existing repository folder is cleaned, so that it does not contain any older files. However, I don't know whether this action is necessary in the context of the build.

Reply:
@oheger-bosch You are absolutely right, but if you open up the lines above, you see that this is already done in the previous step. But you are right, it is better to change it at the first call of these two tools.

Reply:
Ups, I think, now the structure is easier to understand. I overlooked that it were two different repositories used.